processors/jdbc/src/main/java/org/beiter/michael/eaudit4j/processors/jdbc/AbstractJdbcProcessor.java

```diff
@@ -447,10 +447,8 @@ private void persistEvent(final String auditStreamName, final ProcessingObjects
             // will actually be executed, and closing the connection here immediately in the non-error case prevents
             // resource leaks. Repeating the same code below in the finalizer is just a safety net.
             con.setAutoCommit(autoCommit);
-            if (!psEvent.isClosed()) {
```

Comment:
The method ```isClosed()``` is not implemented in all Connection Pools. Some Connection Pools (like C3PO) thows an exception when call it. In this point is almost impossible the psEvent to be closed and I removed.